src/Models/style_base_model.jl

```diff
@@ -38,12 +50,31 @@ Models.variable_types(
     ρw = Geometry.Covariant3Vector{FT},
 )
 
-Models.variable_spaces(::AdvectiveForm) = (
+Models.variable_spaces(::AdvectiveForm, ::AbstractSingleColumnModel) = (
+    ρ = Spaces.FiniteDifferenceSpace{Spaces.CellCenter},
+    uh = Spaces.FiniteDifferenceSpace{Spaces.CellCenter},
+    w = Spaces.FiniteDifferenceSpace{Spaces.CellFace},
+)
```

Comment:
What is this needed for?

Reply:
To follow the default setup of initializing the FieldVectors by zeros, see [here](https://github.com/CliMA/ClimaAtmos.jl/blob/main/src/Models/Nonhydrostatic3DModels/nonhydrostatic_3d_model.jl#L28-L64) and [here](https://github.com/CliMA/ClimaAtmos.jl/blob/main/src/Simulations/simulation.jl#L32) as an example in the 3d model